dvc/remote/local.py

```diff
@@ -76,10 +76,9 @@ def get(self, md5):
 
         return self.checksum_to_path_info(md5).url
 
-    @staticmethod
-    def exists(path_info):
+    def exists(self, path_info):
         assert path_info.scheme == "local"
-        return os.path.lexists(fspath_py35(path_info))
+        return self.repo.tree.exists(fspath_py35(path_info))
```

Comment:
So currently we have a mixup of direct `os.path` and `self.repo.tree` interfaces in our RemoteLOCAL, which is not ideal. We really should work on trees, but for now this is the only thing that was blocking `params diff`, so I've adjusted it. Luckily these are very easy to find with simple tests.